app/src/main/java/com/kickstarter/ui/adapters/ActivityFeedAdapter.java

```diff
@@ -42,6 +49,14 @@ public void takeActivities(final @NonNull List<Activity> activities) {
     notifyDataSetChanged();
   }
 
+  public void takeSurveys(final @NonNull List<SurveyResponse> surveyResponses) {
+    if (surveyResponses.size() > 0) {
+      setSection(SECTION_SURVEYS_HEADER_VIEW, Collections.singletonList(surveyResponses.size()));
+      setSection(SECTION_SURVEYS_VIEW, surveyResponses);
+    }
```

Comment:
we need an `else` to this `if` so that if surveys later load 0 results we will clear the area.

a good test case to have too.